stat.c

```diff
@@ -1600,6 +1740,10 @@ void sum_thread_stats(struct thread_stat *dst, struct thread_stat *src,
 				dst->runtime[l] = src->runtime[l];
 		} else {
 			sum_stat(&dst->clat_stat[0], &src->clat_stat[l], first);
+			if (l == DDIR_READ) {
+				sum_stat(&dst->clat_prio_stat, &src->clat_prio_stat, first);
```

Comment:
How come these don't follow the array forms of the lines below?

Reply:
Since we're only tracking data for reads, I didn't make the the prio stats use the normal DDIR array, since only the reads will ever be populated. Since the other I/O types won't be tracked I didn't think I should be creating an array for them.